src/Checker/PhpVersionChecker/CheckerStrategy/DeployYamlFilesPhpVersionStrategy.php

```diff
@@ -78,15 +78,17 @@ public function check(array $allowedPhpVersions, string $path): CheckerStrategyR
      */
     protected function checkDeployFile(string $fileName, array $deployStructure, array $allowedPhpVersions): CheckerStrategyResponse
     {
-        if (!isset($deployStructure['image']['tag'])) {
+        if (!isset($deployStructure['image']['tag']) && (!isset($deployStructure['image']) || !is_string($deployStructure['image']))) {
```

Comment:
$deployStructure['image']['tag'] and !is_string($deployStructure['image'])) pls check condition